public/heatmap_tooltip.css

```diff
@@ -0,0 +1,27 @@
+.heatmap-tooltip {
+  position: absolute;
+  width: auto;
+  height: auto;
+  padding: 5px;
+  z-index: 150;
+  background-color: #d3d3d3;

```

Comment:
I think this can be changed to match the style in Kibana, which is:
fadeout(@gray-darker, 7%);
gray-darker: #222222